ronn/geopm_report.7.ronn

```diff
@@ -0,0 +1,288 @@
+geopm_report(7) -- geopm summary report file
+============================================
+
+## DESCRIPTION
+
+The GEOPM report is a YAML file containing summarized information
+about runtimes, network usage, energy consumption, and other data
+about an application's behavior on every compute node where it ran.
+The report is broken into sections with totals for each compute node,
+and further divided into sections detailing each application region
+and epoch.  By default, the report will be saved into a file called
+"geopm.report", but this name can be customized with the GEOPM_REPORT
+environment variable; refer to the description for `--geopm-report` in
+**geopmlaunch(1)**.
+
+The application regions and epoch are defined by use of the
+**geopm_prof_c(3)** interface to mark up the user application, or
+through automatic inference of regions based on interposing on the MPI
+or OpenMP interfaces (interposing on OpenMP requires that OMPT is
+enabled at GEOPM compile time, and the `--geopm-ompt-disable` option
+is not provided to `geopmlaunch`). Alternatively, epochs may be
+inserted automatically when `--geopm-record-filter` is used, as
+described in **geopmlaunch(1)**.
+
+## NOTES ON SAMPLING
+
+Most data in the report is derived from PlatformIO signals (described
+in **geopm::PlatformIO(3)**) sampled at a rate determined by the
+Agent::wait() implementation for the active agent (see
+**geopm::Agent(3)**).  Every sample interval, all values of interest
+to the report are read, and the delta values between the last sample
+and current sample are calculated.  This delta is attributed to the
+current region hash for the corresponding domain.  By default,
+synchronous values in the report are aggregated to the board domain,
+so deltas will be attributed to a region only when all active CPUs on
+the compute node are in that region.  User extensions through
+GEOPM_REPORT_SIGNALS may be aggregated to smaller domains.  If any
+CPUs within the domain are in different regions at the time of the
+sample, the delta value will be attributed to the unmarked region.
+Only active CPUs assigned to the application are considered when
+determining the region to attribute values to.
+
```

Comment:
Can you add a note for sync-runtime vs runtime? Sync runtime is sum(delta(time)) for all samples that all active cores were in the same region so it is sample based vs runtime is async wrt GEOPM sampling intervals.